package.json

```diff
@@ -177,6 +177,16 @@
             ]
           },
           "description": "Give an object that's merged into the debug configuration provider"
+        },
+        "mesonbuild.languageServerPath": {
```

Comment:
It is possible that there could be multiple language server implementations in the future. Perhaps these settings could take inspiration from the Python extension because I think it supports multiple?

Reply:
That's implicitly supported. You just have to point it to an executable that `--lsp --stdio` as parameters. Sure not optimal

Reply:
I think this extension having a blessed default (yours) makes the most sense. Then someone could change the setting if another language server ever cropped up.

I don't think any pop up is necessary other than asking do you want to download the latest version of the configured language server.

Reply:
Had an idea: Maybe instead of downloading, setup everything automatically, if a supported distribution is found. That would require root rights, but I'm sure this could work in some way

Reply:
This thread will be obsoleted by https://github.com/JCWasmx86/vscode-meson/pull/3